src/sbt-test/sbt-plugins/simple/test

```diff
@@ -0,0 +1,29 @@
+# See http://eed3si9n.com/testing-sbt-plugins for sbt-scripted test syntax
+# initialize a git repository to make our plugins happy
```

Comment:
Do you need to wipe this directory at the end of the test? I worry it might error if it runs twice without a clean - but I have no idea where this is running, or how scripted starts it. :)